checkov/secrets/plugins/custom_regex_detector.py

```diff
@@ -39,18 +39,17 @@ def __init__(self) -> None:
 
         for detector in detectors:
             try:
-                if detector.get("isMultiline"):
-                    # If prerun exists, we will add it as 'regular detector' (special treat in analyze_line)
-                    if detector.get("prerun"):
-                        self.denylist.add(re.compile('{}'.format(detector["prerun"])))
-                        self.regex_to_metadata[detector["prerun"]] = detector
-                        self.multiline_pattern_by_prerun_compiled[detector["prerun"]] = re.compile('{}'.format(detector["Regex"]))
-                    else:
+                if detector.get("prerun"):
+                    self.denylist.add(re.compile('{}'.format(detector["prerun"])))
+                    self.regex_to_metadata[detector["prerun"]] = detector
+                    self.pattern_by_prerun_compiled[detector["prerun"]] = re.compile('{}'.format(detector["Regex"]))
+                else:
```

Comment:
you can do 
continue instead of the else.. 
it will have less indentation that way

Reply:
done